packages/create-ship-app/src/index.js

```diff
@@ -42,7 +42,7 @@ let deploymentType;
   const dockerComposeFileName = utils.getDockerComposeFileName(apiType, dbType);
   const deploymentFolderNames = utils.getDeploymentFolderNames(deploymentType, apiType, dbType);
   
-  await utils.installServices(projectName, buildType, apiType, dbType, dockerComposeFileName, deploymentFolderNames);
+  await utils.installServices(projectName, buildType, deploymentType.replace(/ /g, '_'), apiType, dbType, dockerComposeFileName, deploymentFolderNames);
```

Comment:
why we need this replace? 🤔

Reply:
shell splits arguments by space. `Digital ocean apps` will be counted as three separate arguments and replace following arguments